sdk/python/feast/infra/local_sqlite.py

```diff
@@ -24,7 +25,7 @@ def _to_naive_utc(ts: datetime):
         return ts.astimezone(pytz.utc).replace(tzinfo=None)
 
 
-class LocalSqlite(Provider):
+class LocalSqlite(InfraProvisioner):
```

Comment:
I'm still a bit confused about what the goal of the `InfraProvisioner` is. It doesn't seem to different from the `Provider`, except in name. Why would it be called `LocalSqlite`?